app/admin/quests/create/page.tsx

```diff
@@ -0,0 +1,1049 @@
+                label="Rewards Title"
+                placeholder="NFT Name"
+              />
+              <Textinput
+                onChange={(e) => {
+                  setQuestInput((prev) => ({
+                    ...prev,
+                    rewards_img: e.target.value,
+                  }));
+                  setNftUri((prev) => ({
+                    ...prev,
+                    image: e.target.value,
+                  }));
+                  setBoostInput((prev) => ({
+                    ...prev,
+                    img_url: e.target.value,
+                  }));
+                }}
+                value={nfturi.image}
+                name="nft_image"
+                label="NFT Image Path"
+                placeholder="NFT Image Path"
+              />
+
+              <Textinput
+                onChange={(e) => {
+                  setNftUri((prev) => ({
+                    ...prev,
+                    description: e.target.value,
+                  }));
+                }}
+                value={nfturi.description ?? ""}
+                name="nft_image"
+                label="NFT Description"
+                placeholder="NFT Description"
+              />
+              <Textinput
+                onChange={handleQuestInputChange}
+                value={questInput.logo}
+                name="logo"
+                label="Issuer Logo"
+                placeholder="Issuer logo"
+              />
+              <div className="flex gap-2 items-center">
+                <p>Boost this quest</p>
+                <Switch
+                  name="Boost this Quest"
+                  value={showBoost}
+                  onChange={() => setShowBoost((prev) => !prev)}
+                />
+              </div>
+            </div>
+            {showBoost ? (
+              <div className="flex flex-col w-full gap-8">
+                <Textinput
+                  onChange={handleBoostInputChange}
+                  value={boostInput.num_of_winners}
+                  name="num_of_winners"
+                  label="Number of winners"
+                  placeholder="Number of winners"
+                />
+                <Dropdown
+                  value={boostInput.token}
+                  backgroundColor="#29282B"
+                  textColor="#fff"
+                  // fullWidth
+                  // sx={{
+                  //   backgroundColor: "#1F1F25",
+                  //   borderRadius: "8px",
+                  // }}
+                  // value={boostInput.token}
+                  // label="Token"
+                  handleChange={(event: SelectChangeEvent) => {
+                    setBoostInput((prev) => ({
+                      ...prev,
+                      token: event.target.value,
+                      token_decimals:
+                        TOKEN_DECIMAL_MAP[
+                          event.target.value as keyof typeof TOKEN_DECIMAL_MAP
+                        ],
```

Comment:
A bit difficult to understand can you clarify or maybe comment to explain ?

Reply:
done